puente-analytics-service/lambdas/etl/index.py

```diff
@@ -1,20 +1,56 @@
-import json
 import os
 import sys; sys.path.append(os.path.join(os.path.dirname(__file__)))
+import json
+import io
+import csv
+from modules.s3Client import s3Client
+from modules.postgres import connection
+
 
-common_req = {
-    "headers": {"Access-Control-Allow-Origin": "*"},
-    "isBase64Encoded": False,
-}
+S3_BUCKET_NAME = os.environ['S3_BUCKET_NAME']
+S3_OBJECT_KEY = os.environ['S3_OBJECT_KEY']
 
 def handler(event, context):
-    print(event)
+    con = connection()
+    cur = con.cursor()
+
+    puenteDataKey = S3_OBJECT_KEY + 'Puente/data/SurveyData/SurveyData.csv'
+
+    csv_file = ''
+    try:
+        csv_object = s3Client().get_object(
+            Bucket=S3_BUCKET_NAME, Key=puenteDataKey)
+        csv_body = csv_object['Body'].read().decode('utf-8')
+        csv_file = io.StringIO(csv_body)
+    except:
+        print('Broken Inside')
+
+    # Parse the CSV data and insert it into the database
+    communities = []
+    reader = csv.DictReader(csv_file)
+    for row in reader:
+        communityName = row['communityname']
+        if communityName not in communities:
+            communities.append(communityName)
+            # Insert data into the 'community_dim' table
+            cur.execute(
+                """
+                INSERT INTO community_dim (name)
+                VALUES (%s)
+                """,
+                (communityName,)
+            )
```

Comment:
@awalsh272 this is just an example to get everything working. Def a placeholder but it works (when there's a community_dim table in the db) 🎊  

just wanted to make sure all the libraries were firing